### DISCUSSION TITLE
Custom LLM Connection Secured by OAuth2

### DISCUSSION THREAD

**Discussion by mundij01** [microsoft/promptflow#2018] [Q&A]
I work for a large company that has a proxy to OpenAI GPT API that does OAuth2 authn/z.  How would I create a custom connection to configure that?  Thanks in advance for the help.

**Comment by brynn-code** [comment]
Hi, sorry for the late response, would you like to elaborate more about your scenario? I'm not sure I get the way of using a proxy that does OAuth2 auth, does that mean you are calling a service to get the token then pass it to OpenAI?
